src/main/kotlin/com/ex_dock/ex_dock/database/product/ProductJdbcVerticle.kt

```diff
@@ -1,6 +1,55 @@
 package com.ex_dock.ex_dock.database.product
 
-import io.vertx.kotlin.coroutines.CoroutineVerticle
+import com.ex_dock.ex_dock.database.connection.Connection
+import io.vertx.core.AbstractVerticle
+import io.vertx.core.eventbus.EventBus
+import io.vertx.kotlin.core.json.json
+import io.vertx.kotlin.core.json.obj
+import io.vertx.sqlclient.Pool
+import io.vertx.sqlclient.Row
+import io.vertx.sqlclient.RowSet
 
-class ProductJdbcVerticle: CoroutineVerticle() {
+class ProductJdbcVerticle: AbstractVerticle() {
+  private lateinit var client: Pool
+  private lateinit var eventBus: EventBus
+
+  override fun start() {
+    client = Connection().getConnection(vertx)
+    eventBus = vertx.eventBus()
+
+    getAllProducts()
+  }
+
+  private fun getAllProducts() {
+    val consumer = eventBus.localConsumer<Any>("process.products.getAll")
```

Comment:
This "consumer" variable name isn't relevant. It shoud be something like "productEventBus"

Reply:
I changed the variable names so it better says what the consumer is for.